subprojects/core/src/main/java/org/gradle/plugin/use/internal/DefaultPluginRequest.java

```diff
@@ -57,6 +57,17 @@ public String getVersion() {
         return version;
     }
 
+    @Nullable
+    @Override
+    public String getArtifact() {
```

Comment:
Can be on the public API of `PluginRequest`. Should be `ModuleVersionSelector`, not `String`.